rust-toolchain

```diff
@@ -1 +0,0 @@
-nightly-2019-05-22
```

Comment:
Please don't remove this, or put the nightly version you're using. It's useful to keep track of which version of the rust compiler this library was written against.

Reply:
Ok, restored and updated to 2019-12-09. Does this file control which toolchain is used for building the package, even if it is being built as a dependency of another package, or is it purely informational? If the former, why is this desirable? I'd have thought that most people would want all their dependency crates built using the same toolchain.